sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/FileFormatWriter.scala

```diff
@@ -104,8 +109,10 @@ object FileFormatWriter extends Logging {
       hadoopConf: Configuration,
       partitionColumns: Seq[Attribute],
       bucketSpec: Option[BucketSpec],
+      statsTrackers: Seq[WriteJobStatsTracker],
       refreshFunction: (Seq[ExecutedWriteSummary]) => Unit,
```

Comment:
do we still need `ExecutedWriteSummary`? After we have the `statsTrackers`, we only need to return `updatedPartitions: Set[String]` to the caller side.